openfisca_france/parameters/prelevements_sociaux/cotisations_regime_assurance_chomage/ags/employeur/ags.yaml

```
@@ -0,0 +1,118 @@
+description: Cotisations employeur de contribution au régime de garantie des salaires (AGS)
+brackets:
+- rate:
+    1974-01-01:
+      value: 0.0005
+    1975-07-01:
+      value: 0.002
+    1976-04-01:
+      value: 0.0025
+    1984-10-01:
+      value: 0.0035
+    1988-01-01:
+      value: 0.0028
+    1989-01-01:
+      value: 0.0024
+    1990-01-01:
+      value: 0.0015
+    1993-01-01:
+      value: 0.0035
+    1996-07-01:
+      value: 0.0025
+    1999-07-01:
+      value: 0.002
+    2000-07-01:
+      value: 0.0015
+    2001-01-01:
+      value: 0.001
+    2002-01-01:
+      value: 0.002
+    2002-07-01:
+      value: 0.003
+    2003-01-01:
+      value: 0.0035
+    2003-09-01:
+      value: 0.0045
+    2005-04-01:
+      value: 0.0035
+    2006-01-01:
+      value: 0.0025
+    2006-07-01:
+      value: 0.0015
+    2008-07-01:
+      value: 0.001
+    2009-04-01:
+      value: 0.002
+    2009-07-01:
+      value: 0.003
+    2009-10-01:
+      value: 0.004
+    2011-04-01:
+      value: 0.003
+    2016-01-01:
+      value: 0.0025
+    2017-01-01:
+      value: 0.002
+    2017-07-01:
+      reference: 
+      value: 0.0015
+  threshold:
+    1974-01-01:
+      value: 0.0
+- rate:
+    1974-01-01:
+      value: 0.0
+  threshold:
+    1974-01-01:
+      value: 4.0
+metadata:
+  ux_name: Taux employeur
+  last_review:
+    2017-07-01
```

j'ai vérifié (vendredi, et tout de suite) ce paramètre qui est à jour du CA du 2021-06-04.
je ne sais pas s'il faut 
- ajouter cette info dans la "reference" du paramètre
- ajouter cette info dans une "reference" qui soit spécifique à la last review
(en effet la référence actuelle pour cette date n'est pas fausse : elle donne bien la référence qui instaure cette valeur).
```suggestion
    2021-07-29
```

---

Vu ! J'ai ajouté l'info dans la reference de 2017 (la derniere value)